include/nuttx/pcnt/pcnt.h

```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ * include/nuttx/pcnt/pcnt.h
```

Comment:
why 2 separate header files?

Reply:
I based rmt driver

Reply:
two header files in rmt also look redundant :)

Reply:
Up to you but we can keep it for now because this driver will be there for temporary.

Reply:
If the mentioned issues are fixed in the near future then I am OK with it. But then it's better not to include it in the next release which will be soon.